src/documents/Contexts.md

```diff
@@ -37,15 +37,6 @@ not available for components at the head of the tree. It is also the most unstab
 it depends on particular naming of members in the tree rather than remaining stable if the tree is
 reorganised. However, it is the only route for matching a particular instance of a component rather
 than matching components by their type or name in general.
-1. The component's "nickname" - which by default is derived from the last path segment of its fully
```

Comment:
[This note is not specific to this line]
Somewhere in this document, there needs to be a link to the ContextAwareness API page.